frontend/src/containers/SurveyPage/SurveyPage.tsx

```diff
@@ -30,61 +74,144 @@ const SurveyPage: React.FC<SurveyPageProps> = ({ user_type }) => {
     // eslint-disable-next-line
   }, []);
 
+  const handleSubmit = async () => {
+    await axiosInstance
+      .post("/api/v1/survey_responses", surveyResponses)
+      .then((response) => {
+        console.log(response);
```

Comment:
I've read that console.logs are usually removed when deploying websites to outside people. Kindly check, thank you.

Reply:
Done